README.md

```diff
@@ -58,6 +59,26 @@ steps:
         from_secret: kubernetes_token
 ```
 
+### Publish
+
+```yaml
+steps:
+  - name: publish
+    image: pelotech/drone-helm3
+    settings:
+      mode: publish
+      chart: ./
+    environment:
+      REGISTRY_URL: 
+        from_secret: registry_url
+      REGISTRY_LOGIN_USER_ID: 
+        from_secret: registry_login_user_id
+      REGISTRY_LOGIN_PASSWORD:
+        from_secret: registry_login_password
+      REGISTRY_REPO_NAME: <helm_repository_name>
+      CHART_VERSION: <version>
```

Comment:
A couple of nitpicks here: 

1. Since `registry_repo_name` and `chart_version` don't need to come from secrets, the `settings` block is a more appropriate spot for them.
1. The README examples should only contain the required params, to minimize the amount of scrolling people have to do.

```suggestion
      registry_repo_name: <helm_repository_name>
      chart_version: <version>
    environment:
      REGISTRY_URL: 
        from_secret: registry_url
      REGISTRY_LOGIN_USER_ID: 
        from_secret: registry_login_user_id
      REGISTRY_LOGIN_PASSWORD:
        from_secret: registry_login_password
```